Handling of invalid escape sequence '\]'

wnark opened the pull request:
To fix this warning:
```
.venv/lib/python3.11/site-packages/wininfparser.py:865
  /opt/cloud-desktop/.venv/lib/python3.11/site-packages/wininfparser.py:865: DeprecationWarning: invalid escape sequence '\]'
```

wnark committed:
Handling of invalid escape sequence '\]'

To fix this warning:
```
.venv/lib/python3.11/site-packages/wininfparser.py:865
  /opt/cloud-desktop/.venv/lib/python3.11/site-packages/wininfparser.py:865: DeprecationWarning: invalid escape sequence '\]'
```

wnark commented:
You may also could use pyupgrade to check that the

arutar reviewed (changes requested):
```
SepRE = re.compile('[^][";=]*(\\]|\\[|"|;|=)?')
```
To get rid of the warning try this code. I suggest replacing it with what I wrote. Please check the code I wrote, this warning does not appear for me. Correct the code in the merge request and I will add a fix

wnark commented:
this warning will only appear once in a brand new python container.

code:`SepRE = re.compile('[^][";=]*(\]|\[|"|;|=)?')` There will be a warning.
code:`SepRE = re.compile('[^][";=]*(\\]|\\[|"|;|=)?')` no warning

wnark committed:
Use the recommended writing style

The writeup has been verified in a brand new docker container without warnings under python 3.11.11

wnark commented:
I've updated it.thanks
@arutar
